src/Umbraco.Web.UI.Client/src/views/common/dialogs/help.html

```diff
@@ -1,13 +1,13 @@
 <div class="umb-panel" ng-controller="Umbraco.Dialogs.HelpController">
     <div class="umb-panel-header">
-        <h1 class="headline">Help</h1>
+        <h1 class="headline"><localize key="sections_help">Help</localize></h1>
         <small class="umb-version">Umbraco version {{version}}</small>
     </div>
 	
 	<div class="umb-panel-body umb-scrollable">
 		<div class="tab-content umb-control-group">
 			<div class="umb-pane">
-				<h5>Help topics for: {{section}}</h5>
+				<h5><localize key="help_helpTopicsFor">Help topics for</localize>: {{section}}</h5>
```

Comment:
@Shazwazza I think ```{{section}}``` could be translated as well by using 
```<localize key="sections_{{section}}">{{section}}</localize>```

E.g. in Danish it will then use these keys: https://github.com/bjarnef/Umbraco-CMS/blob/5bc0c499d8d413e65eeaa6ac49184786e4e5404a/src/Umbraco.Web.UI/umbraco/config/lang/da.xml#L663-L679

It would be great if you just could changes those manually.